static/js/legal/LegalSearch.js

```diff
@@ -9,24 +9,36 @@ const Pagination = require('./Pagination');
 class LegalSearch extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { q: $('#query').val(), from_hit: 0, advisory_opinions: []}
+    const initState = URI.parseQuery(window.location.search);
```

Comment:
Still getting my head around es6, but I thought you weren't supposed to use `const` when you're changing things? Should this be a `let` instead?